dispatches/models/fossil_case/properties/hitecsalt_properties.py

```diff
@@ -0,0 +1,424 @@
+        and Thermochemical Properties
+    (2) Change et al., (2015) Energy Procedia, 69, 779 - 789
+    
+updated on 05/18/2021
+"""
+
+
+# Import Python libraries
+import logging
+# Import Pyomo libraries
+import pyomo.environ as pyo
+from pyomo.environ import (Constraint,
+                           Param,
+                           PositiveReals,
+                           Expression,
+                           RangeSet,
+                           Var,
+                           Reals)
+from pyomo.environ import units as pyunits
+from pyomo.opt import TerminationCondition
+# Import IDAES
+from idaes.core import (declare_process_block_class,
+                        StateBlock,
+                        MaterialBalanceType,
+                        EnergyBalanceType,
+                        StateBlockData,
+                        PhysicalParameterBlock)
+from idaes.core.util.misc import add_object_reference
+from idaes.core.util.initialization import solve_indexed_blocks
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.phases import LiquidPhase
+from idaes.core.components import Component
+from idaes.core.util import get_solver
+import idaes.logger as idaeslog
+
+# Some more inforation about this module
+_author_ = "Naresh Susarla"
+__version__ = "0.0.1"
+
+        
+# Logger
+_log = logging.getLogger(__name__)
+
+# *****************************************************************************
+    
+# *****************************************************************************    
+@declare_process_block_class("HitecsaltParameterBlock")
+class PhysicalParameterData(PhysicalParameterBlock):
+    """
+    Property Parameter Block Class.
+    Contains parameters associated with properties for Hitec Salt.
+    """
+
+    def build(self):
+        """Callable method for Block construction."""
+        super(PhysicalParameterData, self).build()
+        self._state_block_class = HitecsaltStateBlock
+        self._make_params()
+        self.set_default_scaling('flow_mass', 0.1)
+        self.set_default_scaling('temperature', 0.01)
+        self.set_default_scaling('pressure', 1e-5)
+        self.set_default_scaling('enthalpy_mass', 1e-5)
+        self.set_default_scaling('density', 1e-3)
+        self.set_default_scaling('cp_specific_heat', 1e-3)
+        self.set_default_scaling('dynamic_viscosity', 10)
+        self.set_default_scaling('thermal_conductivity', 10)
+        self.set_default_scaling('enthalpy_flow_terms', 1e-6)
+        
+    def _make_params(self):
+        
+        # Add Phase objects
+        self.Liq = LiquidPhase()
+        self.Solar_Salt = Component()
+
+#        Specific heat capacity at constant pressure (cp) coefficients
+#        Cp in J/kg/K
+        cp_param = {1: 5806,
+                    2: -10.833,
+                    3: 7.2413E-3}
+        self.cp_param = Param(RangeSet(3), initialize=cp_param,
```

Comment:
What are the units of these parameters? You should be setting these in the variable discussion, and if they are not the same you will need to create separate Params for each coefficient.

Reply:
addressed